packages/web/src/scss/components/RadioField/index.html

```diff
@@ -1,24 +1,30 @@
 {{#> layout/plain }}
 
-  <div class="docs-FormFieldGrid">
-
-    <label for="radiofield1" class="RadioField">
-      <input type="radio" id="radiofield1" name="example" class="RadioField__input" />
-      <span class="RadioField__label">Radio unselected</span>
-    </label>
-    <label for="radiofield2" class="RadioField RadioField--disabled">
-      <input type="radio" id="radiofield2" name="example" class="RadioField__input" disabled />
-      <span class="RadioField__label">Radio unselected</span>
-    </label>
-    <label for="radiofield3" class="RadioField">
-      <input type="radio" id="radiofield3" name="example" class="RadioField__input" checked />
-      <span class="RadioField__label">Radio selected</span>
-    </label>
-    <label for="radiofield4" class="RadioField RadioField--disabled">
-      <input type="radio" id="radiofield4" name="example2" class="RadioField__input" checked disabled />
-      <span class="RadioField__label">Radio selected</span>
-    </label>
-
-  </div>
+<div class="docs-FormFieldGrid">
+  <label for="radiofield1" class="RadioField">
+    <input type="radio" id="radiofield1" name="example" class="RadioField__input" />
+    <span class="RadioField__label">Radio unselected</span>
+  </label>
+  <label for="radiofield2" class="RadioField RadioField--disabled">
+    <input type="radio" id="radiofield2" name="example" class="RadioField__input" disabled />
+    <span class="RadioField__label">Radio unselected</span>
+  </label>
+  <label for="radiofield3" class="RadioField">
+    <input type="radio" id="radiofield3" name="example" class="RadioField__input" checked />
+    <span class="RadioField__label">Radio selected</span>
+  </label>
+  <label for="radiofield4" class="RadioField RadioField--disabled">
+    <input type="radio" id="radiofield4" name="example2" class="RadioField__input" checked disabled />
+    <span class="RadioField__label">Radio selected</span>
+  </label>
+  <label for="radiofield5" class="RadioField RadioField--isItem">
+    <input type="radio" id="radiofield5" name="example" class="RadioField__input" />
+    <span class="RadioField__label">Radio Item unselected</span>
+  </label>
+  <label for="radiofield6" class="RadioField RadioField--isItem RadioField--disabled">
+    <input type="radio" id="radiofield6" name="example2" class="RadioField__input" checked disabled />
+    <span class="RadioField__label">Radio Item selected disabled</span>
+  </label>
+</div>
```

Comment:
Please present also the selected state so we can always check the design:

<img width="680" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/207590138-34a21f31-2e7d-4c12-abb9-768f30a53955.png">